processmanager/manager_test.go

```diff
@@ -94,7 +94,7 @@ type dummyStackDeltaProvider struct{}
 // GetIntervalStructuresForFile fills in the expected data structure with semi random data.
 func (d *dummyStackDeltaProvider) GetIntervalStructuresForFile(_ host.FileID,
 	_ *pfelf.Reference, result *sdtypes.IntervalData) error {
-	r := rand.New(rand.NewPCG(42, 42))
+	r := rand.New(rand.NewPCG(42, 42)) //nolint:gosec
```

Comment:
Haha, I guess the gosec lints for the v2 rand API simply hadn't been updated?

Reply:
Look like :)